### PR TITLE
chore: add issue template for feature requests

### LINKED ISSUE #841: Update Github Issue template
Currently we only have issue templates for bugs, but there should also be an issue template for feature requests including the following information:
- what area will this influence (e.g. DPF, CI, build, transfer, etc.)
- why is the feature wanted, which requirement is there
- if possible, a (brief!) solution proposal
- type of issue, i.e. "new feature", "improvement", "cleanup"

Checklist:
- assign correct label
- do NOT select a milestone or an assignee!

### PR DESCRIPTION
**Pull request by juliapampus**
## What this PR changes/adds

Adds an issue template for feature requests and provides a link to blank issues.

## Why it does that

To make people's lives easier.

## Linked Issue(s)

Closes #841 

## Checklist

- [x] added relevant details to the changelog? (skip with label `no-changelog`)
- [x] linked GitHub project? (see the section on the right-hand side -->)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+about: Help us with new ideas
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Feature Request
+
+_If you have an idea or an improvement for this project that should be discussed first, please feel
```
Comment (alexandrudanciu) [inline .github/ISSUE_TEMPLATE/feature_request.md:12]:
```suggestion
_If you are missing a feature or have an idea how to improve this project, please feel 
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+assignees: ''
+
+---
+
+# Feature Request
+
+_If you have an idea or an improvement for this project that should be discussed first, please feel 
+free to open up a [discussion](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions/categories/ideas)._
+
+## What Area Will This Influence?
```
Comment (alexandrudanciu) [inline .github/ISSUE_TEMPLATE/feature_request.md:15]:
```suggestion
## Which Areas Would Be Affected?
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+_e.g., DPF, CI, build, transfer, etc._
+
+## Why Is the Feature Wanted?
```
Comment (alexandrudanciu) [inline .github/ISSUE_TEMPLATE/feature_request.md:18]:
```suggestion
## Why Is the Feature Desired?
```

**Review by alexandrudanciu — COMMENTED** [review]
Added some minor improvement suggestions.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Solution Proposal
+_If possible, provide a (brief!) solution proposal._
+
+## Type of Issue
+_i.e., new feature, improvement, cleanup, etc._
+
+## Checklist
+
+- [ ] assigned correct label?
```
Comment (paullatzelsperger) [inline .github/ISSUE_TEMPLATE/feature_request.md:29]:
```suggestion
- [ ] assigned appropriate label?
```
There might not be an exactly matching label.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] assigned project, if necessary? (_see the section on the right-hand side -->_)
```
Comment (paullatzelsperger) [inline .github/ISSUE_TEMPLATE/feature_request.md:30]: I don't think non-committers can do that. Also, does that not happen automatically?

Comment (juliapampus) [reply .github/ISSUE_TEMPLATE/feature_request.md:30]: I agree with you, will remove it.

> Also, does that not happen automatically?

No, it does not, as we could have multiple projects and decided to not have an "open issue" and "backlog" section.

Comment (juliapampus) [reply .github/ISSUE_TEMPLATE/feature_request.md:30]: The automatism takes place as soon as an issue is initially added to the backlog.

**Review by alexandrudanciu — APPROVED** [review]

**Pull request merged**